CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 Change Log
 ==========
 
+Version 2.1.0
+-------------
+
+_2020-10-14_
+
+* Introduce support for rendering across subcompositions. (#104)
+* Fix for bug where layout IDs to skip were queried incorrectly. (#94 – thanks @samruston!)
+* Fix grammatical errors in documentation. (#92 – thanks @androiddevnotesfork!)
+* Upgrade Compose version being compiled and tested against to alpha05. (#106)
```

Comment:
> against to alpha05

=> against alpha05?

Reply:
"Upgrade Compose version (being compiled and tested against) to alpha05."

Reply:
makes sense now but definitely not obvious

Reply:
Removed the extra wording, just saying "Updated compose to x.y.z" instead, like previous changelogs.